fixed typo at readme and srpAnalytics folder in dataQcBmd.py

kimdn opened the pull request:
Please consider to mege this change (please refer my MS chat for detail).

sgosline commented on dataQcBmd.py:
```diff
@@ -117,7 +118,7 @@ def run_morpho_on_file(morph_file,full_devel='full'):
     """
     print ("morpho_input_csv_file_name:" + str(morph_file))
     #to_be_processed/7_PAH_zf_LPR_data_2021JAN11_tall.csv
-    command = "python3 /srpAnalytics/format_morpho_input.py " + \
```
Can you please confirm that this change works in the Docker image? I worry about not having absolute paths.

sgosline commented on dataQcBmd.py:
```diff
@@ -181,11 +182,11 @@ def main():
     if args.morpho=="":
         if args.test_lpr:
             print("Testing LPR code")
-            test_lpr = '/srpAnalytics/test_files/7_PAH_zf_LPR_data_2020NOV11_tall.csv'
```
Same goes for these lines.

sgosline reviewed (commented):
Hi @kimdn these look good, but I have struggled with relative paths in Docker images before, so please confirm that the tests pass (e.g. sh run_tests.sh) before committing.

kimdn commented:
@sgosline 

So, here is what I found/come to know.

Github based unit test (I think that you established?) works fine.
run_tests.sh in srpAnalytics repo works fine.

However, when I ran 
`python3 dataQcBmd.py --test-morpho`
according to README in https://github.com/PNNL-CompBio/srpAnalytics 
it says
```
python3 /srpAnalytics/format_morpho_input.py /srpAnalytics/test_files/7_PAH_zf_morphology_data_2020NOV11_tall.csv devel
python3: can't open file '/srpAnalytics/format_morpho_input.py': [Errno 2] No such file or directory
```
that's why I removed 'srpAnalytics' folder before 'format_morpho_input.py'.

However, once I removed 'srpAnalytics' folder before 'format_morpho_input.py', github based unit tests seem to fail.

When you wrote 
`python3 dataQcBmd.py --test-morpho`
did you check it is working in your laptop?

sgosline commented:
Yes, the readme is incorrect.

kimdn commented:
This PR is now deprecated.
